staging/src/k8s.io/client-go/util/retry/util.go

```diff
@@ -58,14 +58,14 @@ var DefaultBackoff = wait.Backoff{
 //     ...
 //
 // TODO: Make Backoff an interface?
-func RetryOnConflict(backoff wait.Backoff, fn func() error) error {
+func OnError(backoff wait.Backoff, errorFunc func(error) bool, fn func() error) error {
```

Comment:
does this change require unit-test modifications?

you can run this locally in the kubernetes source dir:
```
GO111MODULE=on go test staging/src/k8s.io/client-go/util/retry/...
```

Reply:
Don't think so. 
It just adds renames `RetryOnConfict` to `OnError` and makes `RetryOnConfict` call `OnError` with hardcoded parameter.

Reply:
/ok-to-test

Reply:
RetryOnError would be more consistent, no?

Reply:
Yes, but its against naming convention `retry.OnError` vs `retry.RetryOnError` :man_shrugging: I'm somehow trying to avoid `foo.fooBar` a pattern where package names already tell you what it does. 
But no hard feelings can change if told so.